docs/filestructure.rst

```diff
@@ -0,0 +1,228 @@
+.. _filestructure:
+
+==================================
+File structure example & first app
+==================================
+
+We will deal here with a suggestion how you could structure your files and folders for your individual atk4 project.
+Let's assume you are planning to create at least several pages with some models and views. This example can be expanded
+and modified to your needs and shows just one concept of how to setup an atk4 project.
+
+
+File structure example
+======================
+
+This file structure is a recommendation and no must. It is a best practice example.
+Feel free to experiment with it and find the ideal file structure for your project.
+
+* config
+
+    * db.php
+
+* public_html
+
+    * vendor (contains all needed composer modules - don't touch them)
+
+        * atk4
+
+            * ...
+
+        * autoload.php
+
+        * ...
+
+    * projectfolder
```

Comment:
project the same as vendor all in root

Reply:
projectfolder probably can be changed to src or App, to be in line with others frameworks

Reply:
I think naming should be as simple and as noob understandable as possible.
If something's called "app" I have to research what app means. 

I've investigated many frameworks and I'm annoyed by namings you have to research first. That's why I've chosen this simple approach...

Concerning vendor in root see above...

Reply:
IMHO i think if you define a folder app and in composer.json you put autoload psr-4 pointing there, all the logic must be there.

Reply:
Basically, but as said you first have to research in composer.json. So 1 step more ;)